contribute/create-pull-request.md

```diff
@@ -43,6 +43,10 @@ Pull requests for Redux contributions must:
 - Not contain code that mutates state in reducers or thunks.
 - Not contain code that accesses the reducers state slice directly. Instead, the code should use state selectors to access state.
 
+Pull requests that add/modify unit test (Jest) must:
+
+- Avoid adding snapshots tests, we are removing current ones incrementally
+- If an existing unit test is written in enzyme, migrate it to RTL (React Testing Library), except  if you’re “just” fixing an issue before a major release (i.e v8), then it’s ok to skip migrating the test to RTL
```

Comment:
```suggestion
- If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), except if you’re “just” fixing an issue before a major release (i.e v8), then it’s ok to skip migrating the test to RTL
```

Reply:
Thank you for the review @Clarity-89 🙌🏾  I made the changes :)